decidim-core/lib/decidim/view_model.rb

```diff
@@ -13,15 +13,24 @@ class ViewModel < Cell::ViewModel
     include Decidim::ActionAuthorization
     include Decidim::ActionAuthorizationHelper
     include Decidim::ReplaceButtonsHelper
+    include Cell::Caching::Notifications
 
     delegate :current_organization, to: :controller
 
+    cache :show, if: :perform_caching? do
+      cache_hash
+    end
+
     def current_user
       context&.dig(:current_user) || controller&.current_user
     end
 
     private
 
+    def perform_caching?
+      false
```

Comment:
You can define `cache_hash` in the ViewModel, make it return `nil` by default, and change `perform_caching?` body to `cache_hash.present?` so that you skip  required method 😄

Reply:
Yes thank you!